content/ch-applications/hhl_tutorial.ipynb

```diff
@@ -295,20 +295,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
-    "def create_eigs(matrix, num_ancillae, num_time_slices, negative_evals):\n",
+    "def create_eigs(matrix, num_auxiliary, num_time_slices, negative_evals):\n",
     "    ne_qfts = [None, None]\n",
     "    if negative_evals:\n",
-    "        num_ancillae += 1\n",
-    "        ne_qfts = [QFT(num_ancillae - 1), QFT(num_ancillae - 1).inverse()]\n",
+    "        num_auxiliary += 1\n",
+    "        ne_qfts = [QFT(num_auxiliary - 1), QFT(num_auxiliary - 1).inverse()]\n",
     "\n",
     "    return EigsQPE(MatrixOperator(matrix=matrix),\n",
-    "                   QFT(num_ancillae).inverse(),\n",
+    "                   QFT(num_auxiliary).inverse(),\n",
     "                   num_time_slices=num_time_slices,\n",
-    "                   num_ancillae=num_ancillae,\n",
+    "                   num_auxiliary=num_auxiliary,\n",
```

Comment:
```suggestion
    "                   num_ancillae=num_auxiliary,\n",
```

Unfortunately I think `num_ancillae` is part of the `EigsQPE` constructor, which we can't change.